OWNERS

```diff
@@ -1,12 +1,7 @@
 approvers:
-  - leodido
-  - fntlnz
-  - kris-nova
-  - mfdii
   - leogr
-reviewers:
+emeritus_approvers:
```

Comment:
```suggestion
  - maxgio92
emeritus_approvers:
```

Since @maxgio92 diligently served the Falco project and maintained the https://github.com/falcosecurity/evolution repository, I believe having it on board here will be beneficial.
This is my evaluation criteria.